models/users.ts

```diff
@@ -0,0 +1,18 @@
+import { Schema, model } from 'mongoose';
+
+const userSchema = new Schema({
```

Comment:
Add a TS type like I did for posts and comments in https://github.com/stavby/michlala-fullstack-course/pull/4/commits/3e72e7a741ebbf8f9cb2cffd263ffc0eb3fcf147